conf/views/disclosure/check-your-answers-disclosure.njk

```diff
@@ -30,14 +30,14 @@
           rows: disclosureSummary
         }) }}
 
-          {{ csrf() | safe }}
+        <form action="{{ routes.controllers.DisclosureDetailsController.onPageLoad().url }}" method="get">
```

Comment:
CSRF is only effective when the action is a POST. I double-checked, as here: https://security.stackexchange.com/questions/36671/csrf-token-in-get-request . The reasons I used a form with a method=GET (not a post) are: 1) we don't have to pass content from the disclosure details to the task list, so no need for a form and a POST, and 2) we don't have a standard component to represent a link like a button, so I had to stay with the button. The href directly in the button did not work as I expected, so I reached out for the form with no content.